apps/mobile/src/libs/components/background/components/planet/styles.ts

```diff
@@ -0,0 +1,10 @@
+import { StyleSheet } from "react-native";
+
+const styles = StyleSheet.create({
+	bubble: {
+		borderRadius: 100,
```

Comment:
Let's set the borderRadius as 50% of the planet width, not fixed.

Reply:
border-radius: expect to receive string. I researched and found some solutions [description](https://www.waldo.com/blog/react-native-border-radius). A simple way is to use 999 pixels or const { height, width } = Dimensions.get('window') and set it as half of borderRadius. What way is better to use?